solr/packaging/build.gradle

```diff
@@ -164,48 +155,7 @@ task dev(type: Copy) {
   into devDir
 }
 
-assemble.dependsOn installDist
+tasks.distZip.enabled = false
+tasks.distTar.enabled = false
```

Comment:
This is a hack to keep using the distribution plugin to assemble the distribution, without producing archive files. I'm sure we could have achieved the same with some copy tasks, but that can be done in a later optimization.

Reply:
I'm not really familiar (or comfortable) with the distribution plugin so I can't help you here. My experience with gradle plugins is that the fewer you use, the smaller chances there are of getting headaches in the future.

Reply:
I filed https://issues.apache.org/jira/browse/SOLR-15893 to investigate this later.